tools/fq.wdl

```diff
@@ -8,24 +8,58 @@ version 1.1
 
 task fqlint {
     meta {
-        description: "This WDL task performs quality control on the input FASTQ pairs to ensure proper formatting."
+        description: "This WDL task performs quality control on the input FASTQs to ensure proper formatting."
     }
 
     parameter_meta {
-        read_one_fastq_gz: "Input FASTQ with read one"
-        read_two_fastq_gz: "Input FASTQ with read two"
+        read_one_fastq: "Input FASTQ with read one"
+        read_two_fastq: "Input FASTQ with read two"
+        disable_validator_codes: {
+            description: "Array of codes to disable specific validators",
+            choices: {
+                S001: "Plus line starts with a '+'",
+                S002: "All characters in sequence line are one of 'ACGTN', case-insensitive",
+                S003: "Name line starts with an '@'",
+                S004: "All four record lines (name, sequence, plus line, and quality) are present",
+                S005: "Sequence and quality lengths are the same",
+                S006: "All characters in quality line are between '!' and '~' (ordinal values)",
+                S007: "All record names are unique",
+                P001: "Each paired read name is the same, excluding interleave"
+            }
+        }
+        single_read_validation_level: {
+            description: "Only use single read validators up to a given level",
+            choices: [
+                "low",
+                "medium",
+                "high"
+            ]
+        }
+        paired_read_validation_level: {
+            description: "Only use paired read validators up to a given level",
+            choices: [
+                "low",
+                "medium",
+                "high"
+            ]
+        }
```

Comment:
I didn't look at the fqlint documentation, but these aren't exclusive, right? You can specify `high` and then disable specific validators.

Reply:
I don't think we need to specify in our documentation. I was just making sure.